src/pages/media/index.tsx

```diff
@@ -102,15 +107,29 @@ export default function MediaPage({ mdxContent }: WithMdxResult) {
         </Container>
       </Container>
 
-      <Container gridCols="8" gridGutters="7" flex={true}>
-        <Article title={t('media:press.title')} subtitle={t('media:press.subtitle')}>
-          <TextContent marginTopLarge={true}>
+      <Container paddingTop={false}>
+        <Container gridCols="9" gridGutters="8">
+          <TextContent>
             {mdxContent && <MDXRemote {...mdxContent} components={{ Button: Button }} />}
           </TextContent>
-        </Article>
+        </Container>
       </Container>
     </Layout>
   );
 }
 
-export const getStaticProps = withMdx({ filename: 'media', folder: 'parts' })();
+export const getStaticProps: GetStaticProps = async (props) => {
+  const mdxContent = await loadMdxFile(
+    {
+      filename: 'media',
+      folder: STATIC_PAGES_PATH,
```

Comment:
I'm not sure I understand these changes. Are we applying something similar to #190 for media pages? 😃 That would be great as it would make it easier to add new entries, but how does it work here considering that we have not yet extracted these entries to their own pages?

Reply:
No, all press reviews remain written in the same Markdown file.